CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangel
 - **TrailingStopLossRule** added ability to look back the last x bars for calculating the trailing stop loss
 
 ### Added
+- :tada: **Enhancement** Added floor() and ceil() to Num.class
```

Comment:
This PR was postponed to 0.14, so please update changelog.
I added entry for 0.14 in https://github.com/ta4j/ta4j/pull/491, but it is waiting for review and merge.